server/db/queries/events.ts

```diff
@@ -14,4 +14,21 @@ const getEvents = () => {
     .catch((err) => console.error(err.stack));
 };
 
-export default { getCategories, getEvents };
+const createEvent = (eventObject: any) => {
+  return db
+    .query(`
+INSERT INTO events(name, image, description, size_limit, owner_id, category, lat, lng, start_time, end_time) VALUES
```

Comment:
doesn't the insert items have to be $1, $2....etc to take input from the front end?

Reply:
I was testing SQL queries to ensure that it was going through. Fortunately they are